doc/api/crypto.md

```diff
@@ -674,7 +674,7 @@ const iv = Buffer.alloc(16, 0); // Initialization vector.
 const decipher = createDecipheriv(algorithm, key, iv);
 
 let decrypted = '';
-decipher.on('readable', () => {
+decipher.on('readable', (chunk) => {
```

Comment:
Does the `'readable'` event really pass an argument? Shouldn't this be a variable declared inside the event handler?

Reply:
I've tried to run the code example (it's self sufficient, you can do it too if you want), and it outputs the expected result with my change.

Reply:
Isn't the only thing that this PR is trying to fix the fact that `chunk` is not declared?

Adding it as an argument "fixes" this because now `chunk` is a declared variable, but it makes little sense because `'readable'` does not pass any arguments.